Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -849,8 +897,29 @@ private static float GetInputSystemDefaultDeadZone()
 
         private static float GetGamepadDeadZone(GamepadSlot gamepadSlot)
         {
-            // TODO implement me!
-            return GetInputSystemDefaultDeadZone();
+            // TODO remove me when init is implemented
+            InitializeGamepadConfigs();
+
+            switch (gamepadSlot)
+            {
+                case GamepadSlot.Slot1:
+                case GamepadSlot.Slot2:
+                case GamepadSlot.Slot3:
+                case GamepadSlot.Slot4:
+                case GamepadSlot.Slot5:
+                case GamepadSlot.Slot6:
+                case GamepadSlot.Slot7:
+                case GamepadSlot.Slot8:
+                case GamepadSlot.Slot9:
+                case GamepadSlot.Slot10:
+                case GamepadSlot.Slot11:
+                case GamepadSlot.Slot12:
+                    return s_GamepadConfigs[(int)gamepadSlot].deadzone;
+                case GamepadSlot.All:
+                    return s_GamepadConfigs[0].deadzone; // TODO what should it mean?
```

Comment:
Indeed, feels like its not applicable if this is an aggregated gamepad shouldn't each individual be used for each individual gamepad and aggregation be on top of those processed results?

Reply:
Review fix: shouldn't be all, and passing Any should throw an exception

Reply:
Fixed: changed method to throw exception if All is passed